build/gulpfile.editor.js

```diff
@@ -85,7 +85,7 @@ const extractEditorSrcTask = task.define('extract-editor-src', () => {
 			`lib.webworker.importscripts.d.ts`
 		],
 		redirects: {
-			'vs/base/browser/ui/octiconLabel/octiconLabel': 'vs/base/browser/ui/octiconLabel/octiconLabel.mock',
+			'vs/base/browser/ui/codiconLabel/codiconLabel': 'vs/base/browser/ui/codiconLabel/codiconLabel.mock',
```

Comment:
Maybe this isn't needed anymore. I believe we have the mock for licensing reasons and I believe they are resolved now. @alexandrudima ?

Reply:
@jrieken Good catch. This is a bad change, the standalone editor needs to ship codicons

@misolori please just delete this line.

Reply:
Done